### ISSUE TITLE
No method override for qualifyColumn?

### ISSUE THREAD

**Issue by gms8994** [topclaudy/compoships#2]
Laravel 5.5, PHP 7.1.16

`Sale::class`
```
public function extra()
{
    return $this->hasOne(SaleExtra::class, ['AccountId', 'DealNumber'], ['AccountId', 'DealNumber']);
}
```

`SaleExtra::class`
```
public function sale()
{
    return $this->belongsTo(Sale::class, ['AccountId', 'DealNumber'], ['AccountId', 'DealNumber']);
}
```

Calling out to 
```
Sale::whereHas('extra', function ($query) use ($b) {
    $query->where('a', '=', $b);
})
```

I receive the error below

```
In Str.php line 103:
                                                             
  [ErrorException]                                           
  mb_strpos() expects parameter 1 to be string, array given 
```

With a stacktrace like
```
Exception trace:
 Illuminate\Foundation\Bootstrap\HandleExceptions->handleError() at n/a:n/a
 mb_strpos() at vendor/laravel/framework/src/Illuminate/Support/Str.php:103
 Illuminate\Support\Str::contains() at vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php:260
 Illuminate\Database\Eloquent\Model->qualifyColumn() at vendor/laravel/framework/src/Illuminate/Database/Eloquent/Relations/HasOneOrMany.php:399
 Illuminate\Database\Eloquent\Relations\HasOneOrMany->getQualifiedParentKeyName() at vendor/laravel/framework/src/Illuminate/Database/Eloquent/Relations/Relation.php:207
 Illuminate\Database\Eloquent\Relations\Relation->getRelationExistenceQuery() at vendor/laravel/framework/src/Illuminate/Database/Eloquent/Relations/HasOneOrMany.php:340
 Illuminate\Database\Eloquent\Relations\HasOneOrMany->getRelationExistenceQuery() at vendor/awobaz/compoships/src/Database/Eloquent/Concerns/QueriesRelationships.php:35
 Awobaz\Compoships\Database\Eloquent\Builder->has() at vendor/laravel/framework/src/Illuminate/Database/Eloquent/Concerns/QueriesRelationships.php:131
 Illuminate\Database\Eloquent\Builder->whereHas() at vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php:1470
``

**Comment by topclaudy** [comment]
'whereHas' feature is not yet implemented. This will be added soon.

**Comment by topclaudy** [comment]
Issue fixed in version [1.0.1 ](https://github.com/topclaudy/compoships/releases/tag/1.0.1)
